src/Sylius/Bundle/AdminBundle/Twig/Component/Order/AddressHistoryComponent.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Twig\Component\Order;
+
+use Sylius\Component\Addressing\Model\AddressLogEntry;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Sylius\TwigHooks\LiveComponent\HookableLiveComponentTrait;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+use Symfony\UX\LiveComponent\DefaultActionTrait;
+use Symfony\UX\TwigComponent\Attribute\ExposeInTemplate;
+
+#[AsLiveComponent(
```

Comment:
Actually, is it necessary to have this component live? It's just a read-only view without any interaction. Unless it's needed for `#[ExposeInTemplate]` to work.